markup/asciidocext/convert.go

```diff
@@ -81,17 +80,17 @@ func (a *asciidocConverter) parseArgs(ctx converter.DocumentContext) []string {
 	var cfg = a.cfg.MarkupConfig.AsciidocExt
 	args := []string{}
 
-	if asciidocext_config.AllowedBackend[cfg.Backend] && cfg.Backend != asciidocext_config.Default.Backend {
-		args = append(args, "-b", cfg.Backend)
-	}
+	args = a.appendArg(args, "--backend", cfg.Backend, asciidocext_config.CliDefault.Backend, asciidocext_config.AllowedBackend)
+
+	args = a.appendArg(args, "--doctype", cfg.DocType, asciidocext_config.CliDefault.DocType, asciidocext_config.AllowedDocType)
 
 	for _, extension := range cfg.Extensions {
 		if !asciidocext_config.AllowedExtensions[extension] {
 			a.cfg.Logger.ERROR.Println("Unsupported asciidoctor extension was passed in. Extension `" + extension + "` ignored.")
 			continue
 		}
 
-		args = append(args, "-r", extension)
```

Comment:
The list of arguments can get quite long. Personally, I would prefer to use the short forms because the calls to `asciidoctor` get generated and are not typed in manually by humans.

Reply:
I'd argue it's the opposite: short flags are more useful when written manually, while in scripts/code, I usually prefer the long form because it makes it more obvious for other people reading the code and code is read more often than it is written. Not sure if there is an overall preference in Hugo for short vs long flag format, or if it is decided on a case by case basis. I'm happy to revert this change if it's the latter.